### PR TITLE
Contrib ncview

### PR DESCRIPTION
**Pull request by tloizou**
Contributing ncview, a visual browser for netCDF format files.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by wpoely86** [comment]
looks ready to merge.

**Comment by boegel** [comment]
@wpoely86: if you use `lgtm` or `LGTM` or `style review OK`, this PR will get an extra +1 in users.ugent.be/~kehoste/easybuild-easyconfigs_pr_overview.html (when I update it)

to get a full 3/3 perfect score, the PR needs to pass the tests (ok here), needs an OK w.r.t. style, and needs a working test report

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/n/ncview/ncview-2.1.2-ictce-5.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+dependencies = [
+    ('netCDF-Fortran', '4.2'),
+    ('UDUNITS', '2.1.24'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ncview'],
+    'dirs': [],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncview/ncview-2.1.2-ictce-5.3.0.eb]: @tloizou: missing moduleclass here

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c9788e5c711d7a22a53d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/77d03bcc6b3232a3e363 for a full test report.

**Comment by boegel** [comment]
going in, thanks @tloizou!

**Pull request merged**